src/simulations/paramscan.jl

```diff
@@ -33,7 +34,7 @@ included in the output `DataFrame`.
 function paramscan(parameters::Dict{Symbol,}, initialize;
   n = 1, agent_step! = dummystep,  model_step! = dummystep,
   progress::Bool = true, include_constants::Bool = false,
-  kwargs...)
+  agents_first::Bool=true, kwargs...)
```

Comment:
You've added this here but is it used inside the source of `paramscan`?

Reply:
Its a kwarg. It will be passed to `run!`. Here I just set a default value.

Reply:
Or am I wrong?

Reply:
I think it doesn't need to be added to `paramscan` at all.

Reply:
paramscan will pass all kwargs including this one to run, run will be the one that sets the default value. So yeah - not needed here.